app/assets/stylesheets/components/curriculum_details.scss

```diff
@@ -51,10 +50,13 @@
 
       h3 {
         margin-bottom: .2em;
-      }
-
-      p {
```

Comment:
I removed the p {} because it wasn't necessary anymore.
The new design did not have the paragraph '13 Lessons'.